themes/gitline/gitline.theme.bash

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+. "$BASH_IT/themes/redline/powerline.base.bash"
```

Comment:
You're loading the file from the `redline` theme here, is that on purpose? Shouldn't you load the `powerline.base.bash` file from the `gitline` folder (see below)?

Reply:
Yes I checked the redline theme file now. I actually moved files when I was installing bashit locally. I apologize I did not change the path. I'm using the powerline.base file just like redline.